src/engine/benchmark/logicExpression/logicExpressionEvaluator_bench.cpp

```diff
@@ -0,0 +1,64 @@
+#include <benchmark/benchmark.h>
+#include <logicExpression/logicExpression.hpp>
+
+using namespace std;
```

Comment:
```suggestion
using std::function;
using std::runtime_error;
using std::string;
```

Reply:
Deleted using namespace std and added std:: where needed